src/bundle/Controller/UniversalDiscoveryController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\Controller;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchHit;
+use eZ\Publish\API\Repository\Values\ValueObject;
+use eZ\Publish\Core\REST\Common\Output\Visitor;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class UniversalDiscoveryController extends Controller
+{
+    /** @var LocationService */
```

Comment:
Didn't we agree on FQCN?

Reply:
We did. For unknown reasons, my IDE settings has been reset to defaults. Will update PR with all the FCQN in a moment.